styles.css

```diff
@@ -516,145 +516,153 @@
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
 }
 
 .ul-form {
   list-style-type: none;
   display: flex;
-flex-direction: column;
-align-items: center;
-padding: 0px;
-gap: 24px;
-border: none;
-margin: 0;
-flex: none;
-order: 1;
-flex-grow: 0;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  gap: 25px;
+  border: none;
+  margin: 0;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
 }
 
 .ul-form input {
   border: none;
-border-bottom:1px solid;
-width: 100%;
-
+  border-bottom: 1px solid;
+  width: 100%;
 }
+
 .ul-form label {
   display: none;
 }
 
-.ul-form textarea {
-  border: none;
+.form-name {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+
+  /* or 160% */
+  padding-left: 20px;
+  display: flex;
+  align-items: center;
+
+  /* Theme 3 / Neutral 6 */
+
   color: #979493;
-  width: 100%;
 }
 
-.form-name {
-  font-family: 'Inter';
-font-style: normal;
-font-weight: 400;
-font-size: 15px;
-line-height: 24px;
-/* or 160% */
-padding-left: 20px;
-display: flex;
-align-items: center;
+.form-Email {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
 
-/* Theme 3 / Neutral 6 */
+  /* or 160% */
+  padding-left: 20px;
+  display: flex;
+  align-items: center;
 
-color: #979493;
+  /* Theme 3 / Neutral 6 */
 
+  color: #979493;
 }
 
-.form-Email {
-  font-family: 'Inter';
-font-style: normal;
-font-weight: 400;
-font-size: 15px;
-line-height: 24px;
-/* or 160% */
-padding-left: 20px;
-display: flex;
-align-items: center;
-
-/* Theme 3 / Neutral 6 */
-
-color: #979493;
-}
-
-.input-textarea{
-  font-family: 'Inter';
-font-style: normal;
-font-weight: 400;
-font-size: 15px;
-line-height: 24px;
-/* or 160% */
-display: flex;
-align-items: center;
-height: 114px;
-width: 400px;
-/* Theme 3 / Neutral 2 */
-
-color: #3C3A39;
-}
-
-.submit-form{
-  font-family: 'Inter';
-font-style: normal;
-font-weight: 700;
-font-size: 17px;
-line-height: 24px;
-display: flex;
-align-items: center;
-text-align: center;
-letter-spacing: 0.03em;
-background: #FF6B00;
-flex: none;
-order: 3;
-flex-grow: 0;
-border:none;
-color:white;
-width: 131px;
-height: 48px;
-display: flex;
-flex-direction: row;
-align-items: center;
-padding: 12px;
-gap: 10px;
-}
-.form-bar{
-  border-top: 1px solid #DFE1E6;
-}
-.socialmediaicons-form{
+.textarea-box {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+
+  /* or 160% */
+  padding-left: 20px;
+  display: flex;
+  align-items: center;
+
+  /* Theme 3 / Neutral 6 */
+
+  color: #979493;
+}
+
+textarea {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  padding: 20px;
+  height: 114px;
+  padding-right: 50px;
+  border: none;
+  color: #3c3a39;
+}
```

Comment:
- [ OPTIONAL] I would highly recommend adding a background colour to your `textarea` like in the Figma template to make it visible. 
`background: #FBF8F7;`